_posts/2021-05-12-Subnero-Multi-channel-Modems.md

```diff
@@ -23,11 +23,11 @@ Out-of-the-box, you can use the multi-channel modems as a normal modem or a stan
 
 These devices use spatial diversity combining techniques to decode transmission from remote modems, in real-time. This increases the effective communication range by acting as a spatial diversity receiver.
 
-In addition to the extra receiving channels, the multi-channel modems are also equipped with sensors such as GPS, compass and a depth sensor, for underwater positioning and tracking applications. By combining techniques such as time or phase difference of arrival with the data from these sensors, three dimensional positioning and tracking is also available with the multi-channel modems.
+In addition to the extra receiving channels, the multi-channel modems can also be equipped with sensors such as GPS and compass for underwater positioning and tracking applications. By combining techniques such as time or phase difference of arrival with the data from these sensors, three dimensional positioning and tracking is also available with the multi-channel modems.
```

Comment:
Why is this file updated? This should not be part of this PR.

Reply:
I haven't got a clue what this is doing here. I wasn't there when I requested the review.

Reply:
This is my mistake, when i rebased it to include the fix for pulse link. I will fix it.